### ISSUE TITLE
install script not working

### ISSUE THREAD

**Issue by dragonruler1000** [pterodactyl-installer/pterodactyl-installer#460]
**Describe the bug**
instantly get: bash: /dev/fd/63: No such file or directory

**To Reproduce**
Steps to reproduce the behaviour:
copy the install script download link

1.paste the install script link 
2. hit enter

**Expected behaviour**
the install script should work
**OS (please complete the following information):**

- OS: Ubuntu 20.04.6
- Installation: didn't get that far

**Additional context**
started installing  pterodactyl following the official steps. but only got as far as setting up a user for MariaDB.

**Comment by Houser-Lab** [comment]
Hey dragonruler1000, I was running into this same issue and checked the issues page. I checked the installation video in the readme and it was the first thing it addressed funnily enough.

The solution is to be logged in as root before you run the command. I achieved this either by typing `sudo -i` which simulates a root login in the terminal, or by doing `sudo su`

After logging in as root perform the bash command again, and make sure to remove the sudo as that will cause the problem to occur again.

Hope I helped!
